store/precomputed_key/s3/cli.go

```diff
@@ -0,0 +1,92 @@
+package s3
+
+import (
+	"time"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	EndpointFlagName        = withFlagPrefix("endpoint")
+	CredentialTypeFlagName  = withFlagPrefix("credential-type")
+	AccessKeyIDFlagName     = withFlagPrefix("access-key-id")     // #nosec G101
+	AccessKeySecretFlagName = withFlagPrefix("access-key-secret") // #nosec G101
+	BucketFlagName          = withFlagPrefix("bucket")
+	PathFlagName            = withFlagPrefix("path")
+	BackupFlagName          = withFlagPrefix("backup")
+	TimeoutFlagName         = withFlagPrefix("timeout")
+)
+
+func withFlagPrefix(s string) string {
+	return "s3." + s
+}
+
+func withEnvPrefix(envPrefix, s string) []string {
+	return []string{envPrefix + "_S3_" + s}
+}
+
+// CLIFlags ... used for S3 backend configuration
+// category is used to group the flags in the help output (see https://cli.urfave.org/v2/examples/flags/#grouping)
+func CLIFlags(envPrefix, category string) []cli.Flag {
+	return []cli.Flag{
+		&cli.StringFlag{
+			Name:    EndpointFlagName,
+			Usage:   "endpoint for S3 storage",
+			Value:   "",
+			EnvVars: withEnvPrefix(envPrefix, "S3_ENDPOINT"),
+		},
+		&cli.StringFlag{
+			Name:     CredentialTypeFlagName,
+			Usage:    "The way to authenticate to S3, options are [iam, static]",
+			EnvVars:  withEnvPrefix(envPrefix, "CREDENTIAL_TYPE"),
+			Category: category,
+		},
+		&cli.StringFlag{
+			Name:    AccessKeyIDFlagName,
+			Usage:   "access key id for S3 storage",
+			Value:   "",
+			EnvVars: withEnvPrefix(envPrefix, "ACCESS_KEY_ID"),
+		},
+		&cli.StringFlag{
+			Name:    AccessKeySecretFlagName,
+			Usage:   "access key secret for S3 storage",
+			Value:   "",
+			EnvVars: withEnvPrefix(envPrefix, "ACCESS_KEY_SECRET"),
+		},
+		&cli.StringFlag{
+			Name:    BucketFlagName,
+			Usage:   "bucket name for S3 storage",
+			EnvVars: withEnvPrefix(envPrefix, "BUCKET"),
```

Comment:
Knit - we explicitly set value in some locations but not all

Reply:
[f80fb01](https://github.com/Layr-Labs/eigenda-proxy/pull/146/commits/f80fb01d04f424265e8f04a099a7b2dd52e447b6)